dockerfiles/e2e/entrypoint.sh

```diff
@@ -61,4 +61,29 @@ fi
 
 
 # Launch tests
-npm run $TEST_SUITE
+if [ $TEST_SUITE == "load-test" ]; then
```

Comment:
Probably the "TEST_SUITE" variable should be checked by ```"if [ -z "$TEST_SUITE" ];"``` or another way for avoiding exceptions if variable empty.